OZprivate/rawJS/OZTreeModule/src/tree_state.js

```diff
@@ -41,12 +41,12 @@ class TreeState {
   setup_canvas(canvas) {
     this._canvas = canvas;
     if (this._xp === null) {
-      this._xp = canvas.width/3;
-      this._yp = canvas.height - 200;
+      this._xp = canvas.clientWidth / 3;
```

Comment:
Didn't look like this was doing much since the layout seemed to recentre on load anyway, but thought best keep it up to date